.editorconfig

```diff
@@ -0,0 +1,10 @@
+# see https://editorconfig.org/
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 2
+indent_style = space
+insert_final_newline = true
+trim_trailing_whitespace = true
```

Comment:
Keeping this for now as this will make sure the code has consistent styling in the repo. this config will work with any IDE. ref. https://editorconfig.org/